glommio/src/executor/mod.rs

```diff
@@ -577,58 +579,24 @@ pub enum Placement {
     /// For the `Unbound` variant, the [`LocalExecutor`]s created by a
     /// [`LocalExecutorPoolBuilder`] are not bound to any CPU.
     Unbound,
-    /* MaxSpread,
-     * MaxPack,
+    /// Iterates over `CpuLocation`s in the machine topology with a high degree
+    /// of separation from previous `CpuLocation`s returned by the iterator.
+    /// The order in which items are returned by `MaxSpread` is
+    /// non-deterministic.  If the number of CPUs requested is larger than the
```

Comment:
why not fail in this case?
If we're asking for a hard bind, I think it is fine to fail in the case where asked_cpus > avail_cpus

Reply:
sounds good -- will revise